static/app/views/replays/deadRageClick/selectorTable.tsx

```diff
@@ -197,7 +200,7 @@ export function SelectorLink({
             },
           }}
         >
-          <TextOverflow>{value}</TextOverflow>
+          <TextOverflow>{display}</TextOverflow>
```

Comment:
i'm wondering if it's confusing that this `display` doesn't match the `value` above in the code snippet (the one that shows up on hover). i know the full selector is needed for search to work though, and logically the code snippet is supposed to be copy-able for search. but it might be weird that the display & the hover look different. what do you think?

Reply:
i'm thinking maybe we should have the hover tooltip match what's displayed in the widget. i think the original intent of the tooltip was show the full selector when it's overflowed, so let's make them match

Reply:
ok! we don't need display selector anymore then!!